### ISSUE TITLE
DeprecationWarning from mongodb

### ISSUE THREAD

**Issue by skadefro** [devconcept/multer-gridfs-storage#258]
**Describe the bug**
When used with mongodb 3.6.4 will show depracation waring
```
(node:10148) DeprecationWarning: Listening to events on the Db class has been deprecated and will be removed in the next major version.
```
Mongodb wan't us to use the client object for events and not the DB object
**Environment**
multer-gridfs-storage: 4.2.0
mongodb: 3.6.4

**Expected behavior**
Not showing the warning.
I created a simple PR to fix it

**Comment by intenser** [comment]
Anyone gonna fix that?

**Comment by devconcept** [comment]
This will be fixed on a minor version after 5.0 release. This fix needs to be applied on a version basis or older versions of mongodb will break.

**Comment by devconcept** [comment]
Fixed on https://github.com/devconcept/multer-gridfs-storage/commit/aa3deea3122a5e2cbcbed340f99dc4655d20e41b will be included in the 5.0 version which is planned to be released soon.

**Comment by devconcept** [comment]
Version 5.0.0 is released and the fix is available.
